src/App.vue

```diff
@@ -1,19 +1,21 @@
 <template>
-  <div id="app">
-    <Session />
-    <template>
+  <div>
+    <CookieBar />
```

Comment:
@faboweb i don't know about this lint rule 🐫 for vue components...

Reply:
We can change it, but I don't think it is so bad.

Reply:
whatever the strongest vue recommendation is...